### ISSUE TITLE
Duplicate <script id="documentation_options"> with Sphinx 4

### ISSUE THREAD

**Issue by andersk** [readthedocs/sphinx_rtd_theme#1159]
### Problem

Sphinx 4.0.0 [changed](https://github.com/sphinx-doc/sphinx/commit/a957d6f7108f2ada9f5c9314d2077f6ccba1fe57) the way the `documentation_options` script is inserted, such that we’re now inserting an extra copy of it. This results in invalid HTML (duplicate `id`).

### Reproducible Project

This is reproducible with a totally empty project:

```console
$ pip install Sphinx sphinx_rtd_theme
Collecting Sphinx
[…]
Successfully installed Jinja2-3.0.1 MarkupSafe-2.0.1 Pygments-2.9.0 Sphinx-4.0.2 alabaster-0.7.12 babel-2.9.1 certifi-2021.5.30 chardet-4.0.0 docutils-0.17.1 idna-2.10 imagesize-1.2.0 packaging-20.9 pyparsing-2.4.7 pytz-2021.1 requests-2.25.1 snowballstemmer-2.1.0 sphinx-rtd-theme-0.5.2 sphinxcontrib-applehelp-1.0.2 sphinxcontrib-devhelp-1.0.2 sphinxcontrib-htmlhelp-2.0.0 sphinxcontrib-jsmath-1.0.1 sphinxcontrib-qthelp-1.0.3 sphinxcontrib-serializinghtml-1.1.5 urllib3-1.26.5
$ mkdir docs
$ touch docs/index.rst
$ sphinx-build -C -D html_theme=sphinx_rtd_theme docs build
Running Sphinx v4.0.2
[…]
The HTML pages are in build.
```

#### Error Logs/Results

```console
$ grep documentation_options build/index.html
      <script type="text/javascript" id="documentation_options" data-url_root="./" src="_static/documentation_options.js"></script>
        <script data-url_root="./" id="documentation_options" src="_static/documentation_options.js"></script>
```

#### Expected Results

Only one copy of `<script id="documentation_options">` should be present.

### Environment Info

- Python Version: 3.8.9
- Sphinx Version: 4.0.2
- RTD Theme Version: 0.5.2

**Comment by Blendify** [comment]
This will be fixed with https://github.com/readthedocs/sphinx_rtd_theme/pull/1123

**Comment by Blendify** [comment]
This has been fixed with 380647d1596cec8bda15dc374aba270f884e6de8
